alf/algorithms/ddpg_algorithm.py

```diff
@@ -194,8 +194,10 @@ def __init__(self,
         self._dqda_clipping = dqda_clipping
 
     def predict_step(self, time_step: TimeStep, state, epsilon_greedy=1.):
+        self._actor_network.eval = True
```

Comment:
torch.nn.Module already has function eval() and train() https://pytorch.org/docs/stable/nn.html#torch.nn.Module.eval for this purpose.